src/scss/_mixins.scss

```diff
@@ -36,52 +36,54 @@
 		}
 
 		h1:not(.o-layout__unstyled-element)	{
-			@include oTypographyHeadingLevel1();
+			@include oTypographyHeading($level: 1);
 		}
 
 		h2:not(.o-layout__unstyled-element) {
-			@include oTypographyHeadingLevel2();
+			@include oTypographyHeading($level: 2);
 		}
 
 		h3:not(.o-layout__unstyled-element) {
-			@include oTypographyHeadingLevel3();
+			@include oTypographyHeading($level: 3);
 		}
 
 		h4:not(.o-layout__unstyled-element) {
-			@include oTypographyHeadingLevel4();
+			@include oTypographyHeading($level: 4);
 		}
 
 		h5:not(.o-layout__unstyled-element) {
-			@include oTypographyHeadingLevel5();
+			@include oTypographyHeading($level: 5);
 		}
 
 		h6:not(.o-layout__unstyled-element) {
-			@include oTypographyHeadingLevel6();
+			@include oTypographyHeading($level: 6);
 		}
 
 		aside:not(.o-layout__unstyled-element) {
 			@include _oLayoutRule('left');
 			align-self: flex-start;
 		}
 
+		ol:not(.o-layout__unstyled-element),
 		ul:not(.o-layout__unstyled-element) {
-			@include oTypographyList;
-			@include oTypographyListUnordered;
+			@include oTypographyList();
+		}
+
+		ul:not(.o-layout__unstyled-element) {
+			@include oTypographyList($type: 'unordered', $include-base-styles: false);
 		}
 
 		ol:not(.o-layout__unstyled-element) {
-			@include oTypographyList;
-			@include oTypographyListOrdered;
+			@include oTypographyList($type: 'ordered', $include-base-styles: false);
 		}
 
 		blockquote:not(.o-layout__unstyled-element) {
-			@include oTypographyBlockquote;
-			// Override default browser margin to match that of `oTypographyBody`.
-			// Removes indent from the blockqupte provided by o-typography wrapper.
-			// https://github.com/Financial-Times/o-typography/issues/189
-			margin: 0 0 oSpacingByIncrement(7);
-			p:last-child {
-				margin-bottom: 0;
+			@include oQuoteBase;
+			@include oQuoteStandard;
+			margin-bottom: oSpacingByIncrement(7);
+			cite {
+				@include oQuoteBaseCite();
+				@include oQuoteStandardCite();
```

Comment:
@chee this feels different to our other content mixins because multiple mixins have to be used together. Should the base mixins be private and included in the standard mixin by default?

Reply:
_posed this on the breaking pr https://github.com/Financial-Times/o-quote/pull/42_